examples/wallet/wallet.v.py

```diff
@@ -1,25 +1,46 @@
 # An example of how you can do a wallet in Viper.
 # Warning: NOT AUDITED. Do not use to store substantial quantities of funds.
 
-addrs: address[5]
+# A list of the owners addresses (there are a maximum of 5 owners)
+owners: address[5]
+# The number of owners required to approve a transaction
 threshold: num
+# The number of transactions that have been approved
 seq: num
 
-def __init__(_addrs: address[5], _threshold: num):
+def __init__(_owners: address[5], _threshold: num):
     for i in range(5):
-        if _addrs[i]:
-            self.addrs[i] = _addrs[i]
+        if _owners[i]:
+            self.owners[i] = _owners[i]
     self.threshold = _threshold
 
 def approve(_seq: num, to: address, value: wei_value, data: bytes <= 4096, sigdata: num256[3][5]) -> bytes <= 4096:
```

Comment:
This function approves transactions (potentially sending `value` wei out). How does this wallet gain value to send?